tests/benchmark/snappy/dynamic/smartConnector/PerfRun.conf.template

```diff
@@ -18,44 +24,38 @@ locatorProperties="-conserve-sockets=false"
 serverProperties="-heap-size=3g -memory-size=3g -conserve-sockets=false"
 
 #Machine Setup
-master=localhost
+master=tkhairnar-dell
```

Comment:
Just add a description in the comment ahead of hostname explaining the issues with specifying the master hostname as localhost.

Reply:
Okay.